controller/control.go

```diff
@@ -706,7 +709,8 @@ getCloneStatus:
 	} else if status == "error" {
 		return fmt.Errorf("Replica clone status returned error %s", address)
 	}
-	c.setReplicaModeNoLock(address, types.RW)
+	c.setReplicaModeNoLock(address, types.WO)
```

Comment:
I guess this is not required as the replica is already in WO mode.

Reply:
This is required because we update the status to RO in UpdateVolStatus